scripts/shared/deploy.sh

```diff
@@ -136,6 +136,18 @@ function schedule_dummy_pod() {
     deploy_resource "${RESOURCES_DIR}"/dummypod.yaml "$ns"
 }
 
+function deploy_subm_global_cm() {
+    local ns="submariner-operator"
+    source "${SCRIPTS_DIR}"/lib/deploy_funcs
+
+    echo "Creating the ${ns} namespace..."
+    kubectl create namespace "${ns}" || :
+    echo Setting up submariner global configmap...
+    export NFTABLES="\"${NFTABLES:-false}\""
```

Comment:
```suggestion
    NFTABLES="${NFTABLES:-false}"
```

in line with the fix in the YAML file. Exporting the variable isn’t useful.